notifications/smart_reminder_services.py

```
@@ -0,0 +1,685 @@
+        feedback: Dict
+    ):
+        """
+        تنظیم زمان‌بندی بر اساس بازخورد
+        """
+        if feedback.get('response_type') == 'NO_RESPONSE':
+            # اگر پاسخی نداده، زمان را تغییر دهیم
+            new_hour = (schedule.scheduled_time.hour + 2) % 24
+            new_time = schedule.scheduled_time.replace(hour=new_hour)
+            
+            # ایجاد یادآوری جدید
+            new_schedule = ReminderSchedule.objects.create(
```

Ruff (0.12.2)

Local variable `new_schedule` is assigned to but never used

Remove assignment to unused variable `new_schedule`

(F841)